community/perl-anyevent/APKBUILD

```diff
@@ -4,7 +4,7 @@
 pkgname=perl-anyevent
 _pkgreal=AnyEvent
 pkgver=7.14
-pkgrel=0
+pkgrel=1
```

Comment:
Moving a package doesn't require a `pkgrel` bump.

Reply:
So, how someone will install a package on community repo with dependencies on testing repo, if testing repo has not used by choice? How abuild moves packages to the community repo if I don't forcibly trigger a rebuild?

Reply:
@tanertas Just the move from testing to community already triggers the rebuild. If people do not have testing, the pkgrel bump is not even necessary to get any updates (because they could not have the package installed).